apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/TransientFaultDetector.cs

```diff
@@ -17,10 +17,24 @@
 namespace Google.Cloud.Spanner.Data
 {
     /// <summary>
+    /// Transactions can be retried using the Transient Fault Handling Application Block.
```

Comment:
Perhaps link to docs on the Transient Fault Handling Application Block. I'm not sure what that is.